src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -304,6 +304,13 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             var functionDetails = await WebFunctionsManager.GetFunctionMetadataResponse(listableFunctions, hostOptions, _hostNameProvider);
             result.Add("functions", new JArray(functionDetails.Select(p => JObject.FromObject(p))));
 
+            // TODO discuss more about the proper refactoring for GetHostJsonExtensionsAsync()
```

Comment:
Please create a new issue in this repo for removing/fixing this code, and link to that issue here. E.g. single comment like "TEMP: refactor this code to properly add extensions in all scenarios (#1234)". We can then include all the details in the issue.

Reply:
Thank you for the review. I added issue, and update the comment as you requested! 
https://github.com/Azure/azure-functions-host/issues/7394